mix.exs

```diff
@@ -57,8 +57,9 @@ defmodule ElixirBoilerplate.Mixfile do
       {:corsica, "~> 1.3"},
 
       # Phoenix
-      {:phoenix, "~> 1.6"},
+      {:phoenix, "~> 1.7.0-rc.2", override: true},
```

Comment:
Since we are upgrading to `1.7` why don’t we leverage _functional components_ more instead of collocating heex templates?

We could also use the `attr` and `slot` definition macros at the sametime to gain compile time type checking inside of the html/component semantic.